lib/linguist/languages.yml

```
@@ -1016,6 +1016,16 @@ ChucK:
   codemirror_mode: clike
   codemirror_mime_type: text/x-java
   language_id: 57
+Circom:
+  type: programming
+  ace_mode: java
+  codemirror_mode: clike
+  codemirror_mime_type: text/x-java
```

These don't seem right to me. The `codemirror` options are optional so if in doubt, remove and leave `ace_mode` as `text`:

```suggestion
  ace_mode: text
```